### DISCUSSION TITLE
How to remove `docs` dir

### DISCUSSION THREAD

**Discussion by saihaj** [facebook/docusaurus#4249] [Q&A] [answered]
Is it possible to remove docs directory. I am trying to use docusaurus to host docs for multiple projects in one repo. You can check it out here https://github.com/shabados/docs.  Each folder in `content` directory corresponds to a project. In our case we don't need the `docs` directory nor the `/docs` route since the site is hosted on docs subdomain it just adds noise to the url. To get rid of this route [docs.shabados.com/docs ](https://docs.shabados.com/docs/) I need to remove `docs` dir but when I try that build fails since it is required parameters. How can I get rid of `docs` directory?

related: https://github.com/shabados/docs/issues/21

**Comment by slorber** [comment, marked as answer]
Not sure to understand your goal

You can disable the `docs` plugin using `docs: false` in the preset.

You can also configure the docs plugin route path if you want to change it.
And you can use multiple docs plugins at the same time (one per project) with the multi-instance support.

**Reply by saihaj** [reply]
yes I am using multiple docs plugins one per project. I just wanted to disable the one that comes in by default.
